pytorch/inference/docker/1.12/py3/Dockerfile.graviton.cpu

```diff
@@ -215,7 +215,7 @@ ARG PYTHON_VERSION=3.8.10
 
 ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
 
-RUN pip install --no-cache-dir "sagemaker-pytorch-inference>=2"
+RUN pip install --no-cache-dir sagemaker-pytorch-inference==2.0.11
```

Comment:
Is there a specific reason for pinning it to this version, or, do we want to have the latest patch version of `2.0.*`?

Reply:
This was requested by the AWS contributors of TorchServe as it's stated the SM toolkit version needs to match the TorchServe version. I would prefer not to do this, but this was the request.